cvm/docker-compose.yml

```diff
@@ -22,12 +22,32 @@ services:
       --async-scheduling
     restart: unless-stopped
     healthcheck:
-      start_interval: 1h30m # Allow up to 90 minutes for initial model loading
+      start_period: 1h30m # Allow up to 90 minutes for initial model loading
       test: ["CMD", "curl", "-f", "http://localhost:8000/health"]
       interval: 30s
       timeout: 10s
       retries: 3
 
+  proxy_api:
+      build:
+        context: ./vllm_proxy
+        dockerfile: Dockerfile
+      container_name: proxy_api
+      depends_on:
+        vllm:
+          condition: service_healthy
+      networks:
+        - vllm
+        - proxy_api
+      environment:
+        - BASE_VLLM_URL=http://vllm:8000
+      # No external ports - internal access only
+      expose:
+        - "7000"
+      command: >
```

Comment:
The command is making the image dependent of the docker compose. This should be replaced by the entrypoint in the Dockerfile.